dao/src/main/java/org/thingsboard/server/dao/resource/BaseImageService.java

```diff
@@ -311,7 +309,8 @@ public TbImageDeleteResult deleteImage(TbResourceInfo imageInfo, boolean force)
             }
         }
         if (success) {
-            deleteResource(tenantId, imageId, force);
+            success = deleteResource(tenantId, imageId, force)
```

Comment:
Delete with `force` equal true, we don't need to do all the validation one more time